test/performance/specs/site-editor.spec.js

```diff
@@ -384,6 +385,63 @@ test.describe( 'Site Editor Performance', () => {
 			}
 		} );
 	} );
+
+	test.describe( 'Loading Pages', () => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyfour' );
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyfour' );
+		} );
+
+		const perPage = 20;
+
+		test( 'Run the test', async ( { page, admin, requestUtils } ) => {
+			await Promise.all(
+				Array.from( { length: perPage }, async ( el, index ) => {
+					const { id } = await requestUtils.createPage( {
+						status: 'publish',
+						title: `Page (${ index })`,
+						content: `
+<!-- wp:heading -->
+<p>Hello</p>
+<!-- /wp:heading -->
+<!-- wp:paragraph -->
+<p>Post content</p>
+<!-- /wp:paragraph -->`,
+					} );
+
+					return id;
+				} )
+			);
+
+			const samples = 10;
+			for ( let i = 1; i <= samples; i++ ) {
+				// We want to start from a fresh state each time, without
+				// queries or patterns already cached.
+				await admin.visitSiteEditor();
+
+				const startTime = performance.now();
+
+				await page.getByRole( 'button', { name: 'Pages' } ).click();
```

Comment:
I wonder if we should be using the "list" view. I think maybe using the "table" view would be better and we could set the url explicitly in the `admin.visitSiteEditor` call as it's less fragile to changes we might do to the root page of "pages".

Reply:
list is also kind of tested in the "navigate" metric.

Reply:
Sure I changed it to start with the trash table view then. The reason for this is that we don't want to measure a complete page load.

Reply:
> list is also kind of tested in the "navigate" metric.

Actually, I just checked this test, and it doesn't test loading the list view, it only tests selecting item from the list after it has loaded.